device/bgp/evpn/bgpevpnevisv6.yaml

```diff
@@ -0,0 +1,24 @@
+components:
+  schemas:
+    Bgp.EvpnEvisV6:
+      description: |-
+        This contains a list of different flavors of EVPN like evpn_vxlan or evpn_mpls etc to be configured per Ethernet segment. 
+        Need to instantiate required type of evpn instance as per requirement.
+      type: object
+      properties:
+        choice:
```

Comment:
seperate eviv4,eviv6, bdv4, bdv6,ethernet segment v4 v6 are not required I think,all atrributes in the corressponding v4 and v6 files are same

Reply:
Please check comments below regarding the need of keeping the v4 and v6 objects separate.